static/app/views/releases/thresholdsList/thresholdGroupRows.tsx

```diff
@@ -15,7 +15,16 @@ import {getExactDuration, parseLargestSuffix} from 'sentry/utils/formatters';
 import useApi from 'sentry/utils/useApi';
 import useOrganization from 'sentry/utils/useOrganization';
 
-import {NEW_THRESHOLD_PREFIX} from '../utils/constants';
+import {
+  CRASH_FREE_SESSION_RATE_STR,
+  CRASH_FREE_USER_RATE_STR as _CRASH_FREE_USER_RATE_STR,
+  FAILURE_RATE_STR as _FAILURE_RATE_STR,
```

Comment:
I'm not familiar with this _NAME usage. What is happening here?

Reply:
Ah this just tells linting that it's not being used atm.

Once we utilize them, we can remove the prepended `_`